lib/Github/Api/User.php

```diff
@@ -234,4 +234,23 @@ public function publicEvents($username)
     {
         return $this->get('/users/'.rawurlencode($username).'/events/public');
     }
+
+    /**
+     * List events performed by an authenticated user.
+     *
+     * @link https://docs.github.com/en/rest/reference/activity#list-events-for-the-authenticated-user
+     *
+     * @param string $username
+     * @param int    $page     the page number of the paginated result set
+     * @param int    $perPage  the number of results per page
+     *
+     * @return array
+     */
+    public function events($username, $page = 1, $perPage = 30)
```

Comment:
Also add the `string` typehint to the `$username` parameter and remove the superfluous `@param` docs

Reply:
I added the typehint and I removed completly the @param docs. I also would remove the complete block of comments, since the function looks pretty readable. But I left it to keep consistency with previous methods.